src/scripts.js

```diff
@@ -1,6 +1,29 @@
 const currentUser = new User(userData[0]);
-let firstNameDisplay = document.querySelector('.first-name-section');
+const firstNameDisplay = document.querySelector('.first-name-section');
+const fullNameDisplay = document.querySelector('.full-name');
+const addressDisplay = document.querySelector('.address');
+const emailDisplay = document.querySelector('.email');
+const strideLengthDisplay = document.querySelector('.stride');
+const dailyStepGoal = document.querySelector('.daily-step-goal');
+const friendsList = document.querySelector('.friends');
 
-const displayUsersFirstName = () => {
+
+// const displayUsersFirstName = () => {
+//   firstNameDisplay.innerText = `Hello ${currentUser.provideUsersFirstName()}!`
+// }
+
+const displayAllUserData = () => {
```

Comment:
I think it would be best to have separate commits for individual functions. The query selectors are fine grouped together, but functions I would separate.